### PR TITLE
fix(hw-610): fix-texts

### PR DESCRIPTION
**Pull request by iGroza**
Fixes #

## Proposed Changes

- HW-610: fix provider text

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

**Review by vivalaakam — APPROVED** [review]

File: src/components/transaction-finish.tsx [hunk 1]
```diff
@@ -137,12 +137,12 @@ export const TransactionFinish = observer(
           <Text
             variant={TextVariant.t14}
             color={Color.textBase2}
-            i18n={I18N.transactionConfirmationHAQQ}
+            children={app.provider.name}
```
Comment (devkudasov) [inline src/components/transaction-finish.tsx:140]: Do not pass children as props. Instead, nest children between the opening and closing tags.sonarlint(typescript:S6748)

File: src/components/transaction-finish.tsx [hunk 2]
```diff
@@ -137,12 +137,12 @@ export const TransactionFinish = observer(
...
           />
           <Text
             variant={TextVariant.t14}
             color={Color.textBase2}
-            i18n={I18N.transactionConfirmationHQ}
+            children={`${STRINGS.NBSP}(${app.provider.denom})`}
```
Comment (devkudasov) [inline src/components/transaction-finish.tsx:145]: Do not pass children as props. Instead, nest children between the opening and closing tags.sonarlint(typescript:S6748)

File: src/components/transaction-nft-finish.tsx [hunk 3]
```diff
@@ -110,7 +111,18 @@ export const TransactionNftFinish = ({
 
       <NetworkFee fee={fee.expectedFee} />
 
-      <Spacer />
+      <View style={styles.providerContainer}>
+        <Text
+          variant={TextVariant.t14}
+          color={Color.textBase2}
+          children={app.provider.name}
```
Comment (devkudasov) [inline src/components/transaction-nft-finish.tsx:118]: Do not pass children as props. Instead, nest children between the opening and closing tags.sonarlint(typescript:S6748)

File: src/components/transaction-nft-finish.tsx [hunk 4]
```diff
@@ -110,7 +111,18 @@ export const TransactionNftFinish = ({
...
+        />
+        <Text
+          variant={TextVariant.t14}
+          color={Color.textBase2}
+          children={`${STRINGS.NBSP}(${app.provider.denom})`}
```
Comment (devkudasov) [inline src/components/transaction-nft-finish.tsx:123]: Do not pass children as props. Instead, nest children between the opening and closing tags.sonarlint(typescript:S6748)

**Review by devkudasov — CHANGES_REQUESTED** [review]

File: src/components/transaction-finish.tsx (hunk 2 above)
Comment (iGroza) [reply src/components/transaction-finish.tsx:145]: what difference?

**Review by devkudasov — APPROVED** [review]

**Pull request merged**